### DISCUSSION TITLE
how to plot the Euler angle in the MRD

### DISCUSSION THREAD

**Discussion by 419278981** [mtex-toolbox/mtex#942] [Ask Anything]
hello！
I have got a series of vector in sample coordinate system. My material is a nickel-based superalloy with FCC structure. The data structure is shown in the figure below. How should I draw them on the spherical pole projection in the (111) crystal plane?
![image](https://user-images.githubusercontent.com/32330722/114152012-522c4b00-9950-11eb-9d40-2c3fc98085bc.png)

THX for your help！

**Comment by kilir** [comment]
Hi,
How are your vectirs related to {111}?
Cheers,
Rüdiger
